lib/charms/glauth_utils/v0/glauth_auxiliary.py

```diff
@@ -134,7 +134,7 @@ def _on_auxiliary_requested(self, event: AuxiliaryRequestedEvent) -> None:
 # to 0 if you are raising the major API version
 LIBPATCH = 1
 
-PYDEPS = ["pydantic~=2.5.3"]
+PYDEPS = ["pydantic>=2.5.3"]
```

Comment:
This library comes from https://github.com/canonical/glauth-utils, we would need to update it there if we want to change the dependency as well.

AFAIK that charm only integrates with glauth, so I don't think you will need to use the `glauth_auxiliary` lib. I would suggest to leave this as it is for now.

Reply:
You are absolutely right, my bad.

I think it would still make sense to loose the constraints there, but given that the [glauth-utils](https://github.com/canonical/glauth-utils) lib is a _team internal_ one, I understand it is up to you folks to decide when to do so.